frontends/web/src/components/backups/backup.tsx

```diff
@@ -36,46 +36,51 @@ export interface Backup {
 
 type Props = BackupsListItemProps & TranslateProps;
 
-class BackupsListItem extends Component<Props> {
```

Comment:
eslint complained about this? wow!

what about splitting this commit into a separate PRs? this is massive PR, it's easier to feed it in smaller bits :D

Reply:
It is already split into single commits, see "eslint: refactor components as a pure function" https://github.com/digitalbitbox/bitbox-wallet-app/pull/1011/commits/1fa9d01c66d2f10d2af0dde4bf480a7553c93c1e

Reply:
expect the last commit is a bunch of different eslint fixes